.github/workflows/gh-pages.yml

```diff
@@ -16,9 +16,6 @@ jobs:
       - name: Build prod
         run: yarn build
 
-      - name: Run jest
```

Comment:
Do we want to never run the unit tests in CI now? (the vitest ones)

Reply:
Tests are ran here, but this indeed breaks the hosted code coverage site https://github.com/leather-wallet/extension/blob/b56c27ba5e5f8da85a9582340671aef0ba1c584f/.github/workflows/code-checks.yml#L113-L123

will make an issue to fix, but it's non-critical